components/vac_header_en.js

```diff
@@ -37,15 +37,16 @@ class VacHeaderEn extends Component {
                   <a
                     href="https://www.canada.ca/en.html"
                     className="h6 light-grey"
+                    rel="noopener"
                   >
-                    Canada.ca
+                    <span className="hidden-xs">Canada.ca</span>
```

Comment:
I'm just wondering about the hidden-xs class being added... I'm not sure if hidden-xs exists in our code's css

Reply:
span tag and class is used by parent application with our intention to reuse the parent CSS in future builds, this will align the application code with the parent code base.